virttest/qemu_devices/qdevices.py

```diff
@@ -2686,8 +2690,9 @@ def is_direct_plug(self, device):
         :param device: the QBaseDevice object
         :return: bool value for directly plug or not.
         """
-        not_direct_plug = device.get_param("pcie_direct_plug", "no") == "no"
```

Comment:
I mean here, after get the value, remove it: `device.set_param("pcie_direct_plug", None)`. I don't really test it, just imagine it should work.

Reply:
Ah okay, but `is_direct_plug` will be called twice, so if set it to `None` at the first time, then the second time cannot get the correct value. And I can't guarantee if it will be called more times later.